//
// Podemos pasar de manera opcional el valor error al catch

try {
} catch (error) {}

try {
} catch {
  error
}
